src/main/java/org/mozilla/jss/ssl/SSLAlertDescription.java

```diff
@@ -12,43 +12,37 @@
 public enum SSLAlertDescription {
 
     // see lib/ssl/ssl3prot.h in NSS
-    CLOSE_NOTIFY                    (0),
-    END_OF_EARLY_DATA               (1), // TLS 1.3
-    UNEXPECTED_MESSAGE              (10, SSLProtocolException.class),
-    BAD_RECORD_MAC                  (20, SSLProtocolException.class),
-    DECRYPTION_FAILED               (21, SSLProtocolException.class), // RFC 5246
-    RECORD_OVERFLOW                 (22, SSLProtocolException.class), // TLS only
-    DECOMPRESSION_FAILURE           (30, SSLProtocolException.class),
-    HANDSHAKE_FAILURE               (40, SSLHandshakeException.class),
-    NO_CERTIFICATE                  (41, SSLPeerUnverifiedException.class), // SSL3 only, NOT TLS
-    BAD_CERTIFICATE                 (42, SSLPeerUnverifiedException.class),
-    UNSUPPORTED_CERTIFICATE         (43, SSLPeerUnverifiedException.class),
-    CERTIFICATE_REVOKED             (44, SSLPeerUnverifiedException.class),
-    CERTIFICATE_EXPIRED             (45, SSLPeerUnverifiedException.class),
-    CERTIFICATE_UNKNOWN             (46, SSLPeerUnverifiedException.class),
-    ILLEGAL_PARAMETER               (47, SSLProtocolException.class),
+    CLOSE_NOTIFY(0), END_OF_EARLY_DATA(1), // TLS 1.3
```

Comment:
Wow this is nasty looking.

Reply:
Yeah the `c` auto-formatting is _bad_.

Reply:
This isn't even the worst offender, one of them decided that a variable `camelCase` had to be split into two words `camel` and `case`, causing all sorts of compiler tragedies.

Reply:
ugh. Is this with `uncrustify`?

Reply:
No, Eclipse. I'm thinking at least giving `uncrustify` a go, it can't be any worse than this!